docs/source/integrations.mdx

```diff
@@ -0,0 +1,11 @@
+# Transformers
+
+... TODO: to be filled out ...
```

Comment:
Here we point out to relevant doc sections in transformers / peft / Trainer + explain how these are integrated 

e.g. for transformers state that you can load any model in 8-bit / 4-bit precision, for PEFT, you can use QLoRA out of the box with `LoraConfig` + 4-bit base model, for Trainer: all bnb optimizers are supported by passing the correct string in `TrainingArguments` : https://github.com/huggingface/transformers/blob/abbffc4525566a48a9733639797c812301218b83/src/transformers/training_args.py#L134 

Few references: 

- transformers: https://huggingface.co/docs/transformers/quantization#bitsandbytes
- PEFT: https://huggingface.co/docs/peft/developer_guides/quantization